pkg/upload/react-native-ios.go

```diff
@@ -151,7 +157,13 @@ func ProcessReactNativeIos(
 					bundlePath = possibleBundleFilePath
 					log.Info("Found bundle file at: " + bundlePath)
 				} else {
-					log.Info("No bundle file found at: " + possibleBundleFilePath)
+					possibleBundleFilePath = filepath.Join(buildSettings.ProjectTempRoot, "ArchiveIntermediates/"+scheme+"/BuildProductsPath/Release-iphoneos/main.jsbundle")
```

Comment:
```suggestion
					possibleBundleFilePath = filepath.Join(buildSettings.ProjectTempRoot, "ArchiveIntermediates", scheme, "BuildProductsPath", filepath.Base(buildSettings.BuiltProductsDir), "main.jsbundle")
```
Same sort of idea here too